topostats/run_topostats.py

```diff
@@ -275,13 +275,18 @@ def main(args=None):
             summary_config["df"] = results.reset_index()
             toposum(summary_config)
         else:
-            LOGGER.info(
-                "There are no results to plot, either you have disabled grains/grainstats/dnatracing or there "
-                "have been errors, please check the log for further information."
+            LOGGER.warning(
+                "There are no results to plot, either...\n\n"
+                "* you have disabled grains/grainstats/dnatracing.\n"
+                "* no grains have been detected across all scans.\n"
+                "* there have been errors.\n\n"
+                "If you are not expecting to detect grains please consider disabling"
+                "grains/grainstats/dnatracing/plotting/summary_stats. If you are expecting to detect grains"
+                "please check log-files for further information."
```

Comment:
```suggestion
                " please check log-files for further information."
```

Lack of space causes:

![image](https://user-images.githubusercontent.com/86117496/235128865-5c02e649-ce34-4b82-84a5-37372b20cf3a.png)